Makefile

```diff
@@ -1 +1,3 @@
 -include node_modules/marketplace-gulp/Makefile
+
+TEST_URL?='http://localhost:8674'
```

Comment:
I think the likelihood of this changing after the project has started is pretty low. An advantage to this is that if we move away from using a mock API then we can point these tests against dev or stage if we want.